lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -132,7 +135,7 @@ def main(argv=None):
 
     # Load Cubes
     current_forecast = load_cube(args.forecast_filepath)
```

Comment:
The coefficient cube will have the long name "emos_coefficients" so we can use this to test that we have read in a diagnostic cube for the first arg, e.g.:
```
if current_cube.name() == 'emos_coefficients':
    throw a wobbly.
```